docs/1.getting-started/12.upgrade.md

```diff
@@ -220,6 +221,33 @@ export default defineNuxtConfig({
 
 Please report an issue if you are doing this, as we do not plan to keep this as configurable.
 
+### `clear` and `clearNuxtData` reset `data` in `useAsyncData` and `useFetch`
+
+🚦 **Impact Level**: Minimal
+
+##### What Changed
+
+If you provide a custom `default` value for `useAsyncData`, this will now be used when calling `clear` or `clearNuxtData` and it will bereset to its default value rather than simply unset.
```

Comment:
```suggestion
If you provide a custom `default` value for `useAsyncData`, this will now be used when calling `clear` or `clearNuxtData` and it will be reset to its default value rather than simply unset.
```